R/chrome.R

```diff
@@ -106,8 +110,13 @@ chrome_print = function(
 
   ws = app$ws
 
+  if ((format == 'pdf') & !all(c(missing(selector), missing(box_model), missing(scale))))
```

Comment:
I would use `&&` like other if in the function
```suggestion
  if ((format == 'pdf') && !all(c(missing(selector), missing(box_model), missing(scale))))
```